ios/browser/api/networking/models/extensions/BraveCertificatePoliciesExtensionModel.mm

```diff
@@ -0,0 +1,186 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "BraveCertificatePoliciesExtensionModel.h"
+#if defined(BRAVE_CORE) // Compiling in Brave-Core
+  #import "brave/ios/browser/api/networking/common/brave_certificate_enums.h"
+  #include "brave/ios/browser/api/networking/utils/brave_certificate_ios_utils.h"
+  #include "brave/ios/browser/api/networking/utils/brave_certificate_utils.h"
+#else
+  #import "brave_certificate_enums.h"
+  #include "brave_certificate_ios_utils.h"
+  #include "brave_certificate_utils.h"
+#endif
+
+#if defined(BRAVE_CORE) // Compiling in Brave-Core
+  #include "third_party/boringssl/src/include/openssl/x509.h"
+  #include "third_party/boringssl/src/include/openssl/x509v3.h"
+#else
+  #include <openssl/x509.h>
+  #include <openssl/x509v3.h>
+#endif
+
+
+@implementation BraveCertificatePolicyInfoQualifierNoticeExtensionModel
+- (instancetype)init {
+  if ((self = [super init])) {
+    _organization = [[NSString alloc] init];
+    _noticeNumbers = @[];
+    _explicitText = [[NSString alloc] init];
+  }
+  return self;
+}
+
+- (void)setOrganization:(NSString*)organization {
+  _organization = organization;
+}
+
+- (void)setNoticeNumbers:(NSArray<NSString*>*)noticeNumbers {
+  _noticeNumbers = noticeNumbers;
+}
+
+- (void)setExplicitText:(NSString*)explicitText {
+  _explicitText = explicitText;
+}
```

Comment:
Why not just redefine these properties as `readwrite` in a empty category in this file? (same for other classes in this file)